roles/openshift-applier/tasks/pre-check.yml

```diff
@@ -12,42 +12,34 @@
     - name: "Retrieve oc client version"
       shell: "{{ item }}"
       ignore_errors: true
-      register: oc_vers_check
+      register: client_vers_check
       with_items:
-        - oc version -o yaml
-        - oc version
+        - "{{ client }} version -o yaml"
+        - "{{ client }} version"
 
     # Block to handle oc command output
     # - only proceed if the oc command returned any output
     - block:
 
-        - name: "Filter out just the oc version number (v4.X)"
+        - name: "Filter out the {{ client }} version number"
           set_fact:
-            oc_version: "{{ (oc_vers_check.results[0].stdout | from_yaml).clientVersion.major }}.{{ (oc_vers_check.results[0].stdout | from_yaml).clientVersion.minor}}"
-          when:
-            - oc_vers_check.results[0].stdout != ""
-
-        - name: "Filter out just the oc version number (v3.X)"
```

Comment:
@etsauer did you intentionally remove this block? It was here to keep the version checks work with oc v3.x as well as oc v4.x. By removing this, it looks to only work consistently for oc v3.x and not v4 (and probably same for kubectl as well. 

Sample output from `kubectl`
```
./kubectl version
Client Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.4+7bd2e5b", GitCommit:"7bd2e5b", GitTreeState:"clean", BuildDate:"2019-05-19T23:52:43Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

Reply:
@oybed I updated the regular expression to work with ALL versions of either oc or kubectl. I tested it with kubectl and oc 3.11 and 4.1 and all of them work fine for me.

Reply:
@etsauer it's working for v4 as it's picking up the `GitVersion:"v.....",` content. IMHO this seems fragile and we should be using the Major/Minor fields since they are there (and probably more meant for the purpose).